bpf/Makefile

```diff
@@ -149,11 +150,9 @@ HOST_OPTIONS = $(LXC_OPTIONS) \
 	-DENABLE_IPV4:-DENABLE_IPV6:-DENCAP_IFINDEX:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DENABLE_DSR_HYBRID:-DENABLE_PREFILTER:-DENABLE_HOST_FIREWALL: \
 	-DENABLE_IPV4:-DENABLE_IPV6:-DENCAP_IFINDEX:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DENABLE_DSR_HYBRID:-DENABLE_PREFILTER:-DENABLE_SESSION_AFFINITY:-DENABLE_HOST_FIREWALL:
 
-MAX_HOST_OPTIONS = $(MAX_BASE_OPTIONS) -DENCAP_IFINDEX=1 -DENABLE_HOST_FIREWALL=1
+MAX_HOST_OPTIONS = $(MAX_BASE_OPTIONS) -DENCAP_IFINDEX=1
 ifneq ("$(KERNEL)","49")
-MAX_HOST_OPTIONS += -DENABLE_HOST_SERVICES_UDP=1 -DENABLE_HOST_SERVICES_TCP=1 \
-	-DENABLE_NODEPORT=1 -DENABLE_NODEPORT_ACCELERATION=1 -DENABLE_EXTERNAL_IP=1 -DENABLE_DSR=1 \
-	-DENABLE_DSR_HYBRID=1 -DENABLE_SESSION_AFFINITY=1
+MAX_HOST_OPTIONS += -DENABLE_EGRESS_GATEWAY=1
```

Comment:
`ENABLE_EGRESS_GATEWAY` is a v1.10 feature.

Reply:
The commit message probably also needs to be updated accordingly.

Reply:
:man_facepalming: